src/vs/workbench/contrib/comments/browser/simpleCommentEditor.ts

```diff
@@ -145,7 +145,7 @@ export class SimpleCommentEditor extends CodeEditorWidget {
 export function calculateEditorHeight(parentEditor: LayoutableEditor, editor: ICodeEditor, currentHeight: number): number {
 	const layoutInfo = editor.getLayoutInfo();
 	const lineHeight = editor.getOption(EditorOption.lineHeight);
-	const contentHeight = (editor._getViewModel()?.getLineCount()! * lineHeight) ?? editor.getContentHeight(); // Can't just call getContentHeight() because it returns an incorrect, large, value when the editor is first created.
+	const contentHeight = (editor._getViewModel()?.getLineCount()! * lineHeight); // Can't just call getContentHeight() because it returns an incorrect, large, value when the editor is first created.
```

Comment:
@alexr00 The `?? editor.getContentHeight()` call was not reachable. Please make sure this change looks correct as I'm a little unsure what the intent was

Reply:
I think the intent was to check for `0`, but if the right of the `??` was unreachable then this is the correct change.